doc/language_guide.md

```diff
@@ -328,6 +330,39 @@ WITH
 USING TreeExplainer;
 ```
 
+## Evaluation Syntax
+
+A SQLFlow prediction statement consists of a sequence of select, predict, and using clauses.
+
+```sql
+SELECT select_expr [, select_expr ...]
+FROM table_references
+  [WHERE where_condition]
+  [LIMIT row_count]
+TO EVALUATE model_table_reference
+[WITH
+  attr_expr [, attr_expr ...]]
+INTO evaluate_result_table;
+```
+
+The select statement syntax is the same as the select statement syntax in the training syntax. When doing the evaluation, you should use a dataset that is not used for training to evaluate the model's performance when dealing with "new" data. The evaluation result will be saved in `evaluate_result_table` when the evaluation job completes.
```

Comment:
The SELECT statement before `TO EVALUATE` must generate the same data schema as that used in training.  Please be aware that, by the machine learning theory, you are not supposed to use the training dataset for evaluation.  If you do that, you are likely to see a very good evaluation result, which indeed, doesn't tell much useful information.  Please write a SELECT statement as the prefix that generates different dataset but have the same schema.

The INTO clause names the table for saving evaluation results, which, in this example, is `evaluate_result_table`.

Reply:
Thanks for the hints. Done and updated.